js/scorm/cookieLMS.js

```diff
@@ -0,0 +1,187 @@
+import Adapt from 'core/js/adapt';
+import Cookies from 'libraries/js-cookie.js';
+
+/** Start the mock API if window.ISCOOKIELMS exists and isn't null */
+export const shouldStart = (Object.prototype.hasOwnProperty.call(window, 'ISCOOKIELMS') && window.ISCOOKIELMS !== null);
+
+/** Store the data in a cookie if window.ISCOOKIELMS is true, otherwise setup the API without storing data. */
+export const isStoringData = (window.ISCOOKIELMS === true);
+
+export function createResetButton() {
+  $('body').append($('<style id="spoor-clear-button">.spoor-reset-button { position:fixed; right:0px; bottom:0px; } </style>'));
+  const $button = $('<button class="spoor-reset-button btn-text">Reset</button>');
+  $('body').append($button);
+  $button.on('click', e => {
+    if (!e.shiftKey) {
+      Cookies.remove('_spoor');
+      alert('SCORM tracking cookie has been deleted! Tip: shift-click reset to preserve cookie.');
```

Comment:
should we return here after the alert?

Reply:
We want to route the user in both cases (holding shift simulates a re-launch as the tracking data is preserved).